### DISCUSSION TITLE
Experiments with structured config

### DISCUSSION THREAD

**Discussion by mbignotti** [facebookresearch/hydra#2550] [Q&A]
Hi! 
I'm not sure if it has already been asked, but I haven't found anything related in discussions and/or issues (apologies for possible duplicate question). 
Is it possible to replicate the experiment configuration pattern (described [here](https://hydra.cc/docs/patterns/configuring_experiments/)) with structured configs? If yes, can I ask to give a very brief example? 

Thanks a lot.

**Comment by omry** [comment]
Yes.
Generally everything you can do with yaml files you can also do with pure Structured Configs.
Just try to translate the examples to data classes.
In particular, the package can be used when adding the dataclasses to the config store and the defaults list should be in the dataclass(es).
